### PR TITLE
components

### PR DESCRIPTION
**Pull request by allamalitska**
Added a new component without any functionality.
Screenshot 2021-12-26 at 23 19 56

### CODE REVIEW

File: src/components/button.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
```
Comment (dandersonstack) [inline src/components/button.js]: rename this file to be the name of the component.
We use CamleCase.js for react components.
(other people use .jsx but not us, cause .js is good)
```ClickCounter.js```
```button.js```

File: src/components/button.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+export default function CountClicks(){
```
Comment (dandersonstack) [inline src/components/button.js]: I THINK ClickCounter makes more sense.

File: src/components/button.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    return(
+    <button> Click me!
+
+    </button>)
+}
```
Comment (dandersonstack) [inline src/components/button.js]: should have new line.

File: src/components/button.js (hunk 2 above)
Comment (dandersonstack) [inline src/components/button.js]: I prefer to have any export at the bottom.

better delcare your component.

const ClickerCounter = ()=>



export default ClickerCounter

export {
anyOtherFunctions
}

File: src/components/button.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+export default function CountClicks(){
+    return(
+    <button> Click me!
```
Comment (dandersonstack) [inline src/components/button.js]: missing the p tag with the counter representation.

https://reactjs.org/docs/hooks-intro.html
 (please use hooks)

**Review by dandersonstack — COMMENTED** [review]
great Job, 
you knocked your first REAL pr out of the park.

I liked you using the react built in <button/> class.

(it is connected DIRECLTY to HTML 5 <button/>

aka. use use <button/>

and it will pass along props to an <html 5> <button/>
which will then render ot to Dom.

this is an exmple of data abstraction.

File: src/components/button.js (hunk 2 above)
Comment (allamalitska) [reply src/components/button.js]: you are right , fixed it

File: src/components/button.js (hunk 3 above)
Comment (allamalitska) [reply src/components/button.js]: fixed it.. sorry, should remember it

File: src/components/button.js (hunk 2 above)
Comment (allamalitska) [reply src/components/button.js]: did it

File: src/components/button.js (hunk 4 above)
Comment (allamalitska) [reply src/components/button.js]: Fixed!

**Comment by allamalitska** [comment]
Love you! Very appreciate, how much you do for me.

**Pull request merged**